src/SQLQueryStress/SQLQueryStress.csproj

```diff
@@ -19,7 +19,7 @@
     <ManifestKeyFile>SQLQueryStress_TemporaryKey.pfx</ManifestKeyFile>
     <IsWebBootstrapper>false</IsWebBootstrapper>
     <ApplicationIcon>Resources\QueryStressIcon.ico</ApplicationIcon>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>

```

Comment:
Why

Reply:
This commit was intended for my fork but I then decided to do the push request, just in case you wanted the export.
4.6 is an in-place update from 4.0, why not?why not update the framework to get better garbage collection, features and overall improvements?

Reply:
You are already getting that if you have a newer .NET version installed, but this will force users with lower versions to upgrade.